1080 move licence field from dataset to distribution

kastger opened the pull request:
- Remove `licence` field from `Dataset` model.
- Add `licence` field to `DatasetDistribution`.
- Make custom migration to add existing licence to distributions that belong to the dataset.
- Update form fields.

karina-klinkeviciute commented on scripts/geoportal_import.py:
```diff
@@ -138,14 +139,14 @@ def main():
-    if results is not None and results.get('numberOfRecordsMatched'):
-        total_results = int(results.get('numberOfRecordsMatched'))
+    if results is not None and results.get("numberOfRecordsMatched"):
+        total_results = int(results.get("numberOfRecordsMatched"))
```
čia koks nors automatinis įrankis performatavo? Šiaip, kažkada kalbėjau su Mantu, tai jis sakė už tai, kad žmogiškas tekstas, kaip realūs žodžiai, turėtų būti dvigubose kabutėse, o įvairūs techniniai dalykai - viengubose. Pagal tą čia turėtų būti viengubos. Nors man visai atrodo ok visur dvigubos - kad nereikėtų papildomo darbo galvoti, kur dėti viengubas, kur dvigubas - o ir dvigubos įprasčiau.

kastger replied:
Padiskutuosiu su Mantu, bet aš irgi manau, kad geriau visur dvigubos visur.

kastger committed:
1080 Move `licence` field from `Dataset` model

- Remove `licence` field from `Dataset` model.
- Add `licence` field to `DatasetDistribution`.
- Make custom migration to add existing licence to distributions that belong to the dataset.
- Update forms.

karina-klinkeviciute commented on vitrina/datasets/models.py:
```diff
@@ -186,15 +186,8 @@ class Dataset(TranslatableModel):
-        verbose_name=_("Licenzija"),
-    )
-
+    status = models.CharField(max_length=255, choices=STATUSES, default=UNASSIGNED)
```
čia antrą kartą `status? Jau yra, dviem eilutėm žemiau.

karina-klinkeviciute commented:
Kaip supratau pagal užduotį, šitas: `distribution_conditions` taip pat turėtų būti perkeltas: "Tas pats galioja ir naudojimo sąlygų laukams."

karina-klinkeviciute commented on vitrina/resources/models.py:
```diff
@@ -168,6 +169,15 @@ class DatasetDistribution(TranslatableModel):
+        db_column="licence",
+        blank=False,
+        null=True,
+        verbose_name=_("Licenzija"),
```
turi būti "Licencija": https://rasyba.lietuviuzodynas.lt/Licencija

karina-klinkeviciute commented on vitrina/resources/models.py:
```diff
@@ -291,6 +301,14 @@ def is_previewable(self):
         )
 
+    def get_license_url(self):
+        url = (
```
kodėl čia taip?

karina-klinkeviciute reviewed (changes requested):
reikėtų padaryti ir šią dalį: Tas pats galioja ir naudojimo sąlygų laukams.

kastger commented:
https://github.com/atviriduomenys/katalogas/issues/1621
